app/utils/feedback-helper.ts

```diff
@@ -0,0 +1,31 @@
+import { ApolloClient, gql } from "@apollo/client"
+import crashlytics from "@react-native-firebase/crashlytics"
+import { FeedbackSubmitDocument } from "@app/graphql/generated"
+
+gql`
+  mutation feedbackSubmit($input: FeedbackSubmitInput!) {
+    feedbackSubmit(input: $input) {
+      errors {
+        message
+      }
+      success
+    }
+  }
+`
+
+export const sendFeedback = async (
```

Comment:
you should use `useSendFeedbackMutation` instead. this feedback helper boilerplate is not needed